docs/src/pages/components/buttons/ButtonSizes.tsx

```diff
@@ -1,67 +1,57 @@
 import * as React from 'react';
-import { createStyles, Theme, makeStyles } from '@material-ui/core/styles';
-import Button from '@material-ui/core/Button';
+import { experimentalStyled as styled } from '@material-ui/core/styles';
+import MuiButton from '@material-ui/core/Button';
 import IconButton from '@material-ui/core/IconButton';
 import DeleteIcon from '@material-ui/icons/Delete';
 import ArrowDownwardIcon from '@material-ui/icons/ArrowDownward';
 
-const useStyles = makeStyles((theme: Theme) =>
-  createStyles({
-    margin: {
-      margin: theme.spacing(1),
-    },
-  }),
+const Button = styled(MuiButton)(
```

Comment:
Use a CSS selector instead of customizing the button directly

```suggestion
```

Reply:
Yes, thanks for the review, it is corrected. Use `& button` instead